pulpcore/app/models/task.py

```diff
@@ -287,11 +287,11 @@ def current():
             pulpcore.app.models.Task: The current task.
         """
         try:
-            task_id = get_current_job().id
+            task_job_id = get_current_job().id
```

Comment:
`task_job_id` was confusing the first time I read it.  Recommend `job_id` would be better.

Reply:
+1 on this